src/main/java/lotto/LottoCustomer.java

```diff
@@ -11,13 +13,14 @@
 
     public static void main(String[] args) {
         int price = InputView.requestPrice();
+        List<LottoNums> lottoNumsList = InputView.requestManualNumsList().stream().map(LottoNums::of).collect(Collectors.toList());
```

Comment:
manual로 입력받은 lottoNums라는걸 알 수 있는 변수명으로 지정해주시면 좋을 것 같네요!

```suggestion
        List<LottoNums> manualLottoNumsList = InputView.requestManualNumsList().stream().map(LottoNums::of).collect(Collectors.toList());
```